### PR TITLE
Updated package versions that have reported vulnerabilities

### PR DESCRIPTION
**Pull request by shaidar**
#### What's this PR do?
Ran [safety](https://pyup.io) against the requirements.txt file and that reported vulnerabilities in three packages. Based on that input, updated package versions that have reported vulnerabilities.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -28,7 +28,7 @@ certifi==2017.4.17        # via requests
 cffi==1.10.0              # via cryptography
 chardet==3.0.4            # via requests
 colorama==0.3.9           # via mondrian
-cryptography==2.1.4
+cryptography==2.3.1
```
Comment (mbertrand) [inline requirements.txt]: Running `pip-compile --output-file requirements.txt requirements.in` also adds `appnope==0.1.0            # via ipython` above.

Comment (mbertrand) [reply requirements.txt]: Never mind about that since it's only for ipython.

**Comment by mbertrand** [comment]
I think once PR #693 is merged into master, and this is rebased on master, the travis errors should go away.

**Comment by pdpinch** [comment]
@shaidar can you rebase?

**Pull request merged**